packages/syft/tests/syft/core/tensor/smpc/smpc_tensor_test.py

```diff
@@ -299,3 +299,34 @@ def test_swapaxes() -> None:
     exp_res = value.swapaxes(0, 1)
 
     assert (res == exp_res).all()
+
+
+def test_pos() -> None:
```

Comment:
shouldn't a test of "pos()" test that it properly converts negative numbers into positive ones?

(also - is this possible without comparison? seems like comparison has to come first)